flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -316,6 +316,11 @@
 	 */
 	public static final String TASK_MANAGER_REFUSED_REGISTRATION_PAUSE = "taskmanager.refused-registration-pause";
 
+	/**
+	 * The config parameter defining the root directories for storing file-based state for local recovery.
+	 */
+	public static final String TASK_MANAGER_LOCAL_STATE_ROOT_DIR_KEY = "taskmanager.local-state-root.dir";
```

Comment:
Using `ConfigConstants` is discouraged. Please introduce a `ConfigOption` if not already done in a later commit.

Reply:
👍